README.md

```diff
@@ -93,6 +93,8 @@ That means it should be compatible with sklearn pipelines, wrappers, and so fort
 
 In addition, Brush provides functionality that allows you to feed in more complicated data types than just matrices of floating point values. 
 
+<!-- TODO: update these examples (at least check they still work) -->
```

Comment:
I am going to update it so the examples work.
Just an observation: I renamed it to pybrush instead of brush, because Joe said (a few months ago) that there was already a package named brush!

Reply:
the python package is already called pybrush https://github.com/cavalab/brush/blob/f54104ff12fb9079c90fabe16defad13f751b8c9/setup.py#L109